catalogue/webapp/pages/work.js

```diff
@@ -65,6 +93,9 @@ export const WorkPage = ({ work }: Props) => {
       }
       if (manifestData && manifestData.manifests) {
         setChildManifestsCount(manifestData.manifests.length);
+        setFirstChildManifest(
+          await getFirstChildManifest(manifestData.manifests)
```

Comment:
Assuming this sets the value to a Promise?
The state isn't typed, so hard to tell.